docs/providers/email.md

```diff
@@ -31,8 +31,9 @@ You can override any of the options to suit your own use case.
 
 ## Configuration
 
-1. You will need an SMTP account; ideally for one of the [services known to work with `nodemailer`](http://nodemailer.com/smtp/well-known/).
-2. There are two ways to configure the SMTP server connection.
+1. NextAuth no longer includes `nodemailer` as a dependency, so you'll need to install it yourself if you want to use the Email Provider. Run `npm install nodemailer` or `yarn install nodemailer`.
```

Comment:
let's reword this a bit:

- the official name is NextAuth.js
- let's remove the "no longer part"
- `yarn add`